resources/views/docs/_structures/news_post.md

```diff
@@ -0,0 +1,29 @@
+## NewsPost
+
+Field        | Type                    | Description
+-------------|-------------------------|------------
+id           | number                  | |
+author       | string                  | |
+edit_url     | string                  | Link to the file view on GitHub.
+first_image  | string?                 | Link to the first image in the document.
+published_at | [Timestamp](#timestamp) | |
+updated_at   | [Timestamp](#timestamp) | |
+slug         | string                  | Unique identifier made by joining the publish date and title.
```

Comment:
The relation to publish date and title isn't actually in the code. It's just the filename used in github